rust-src/wallet_library/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Added
 
+- Added trait `WalletCheck` with method `satifies_wallet_restrictions`, and implemented it web3IdProofInput, request and statements.
```

Comment:
"and implemented it ..."

Something is missing. Also quote `Web3IdProofInput` and capitalize correctly please.